components/appmgt/org.wso2.carbon.appmgt.usage/org.wso2.carbon.appmgt.usage.publisher/src/main/java/org/wso2/carbon/appmgt/usage/publisher/APPMgtUiActivitiesBamDataPublisher.java

```diff
@@ -232,20 +220,18 @@ public void publishUserActivityEvents(String action, String item,
 				}
 			} else {
 				// Write directly to DB
-				AppMDAO.saveStoreHits(appId.trim(), userId.trim(), tenantId,
-						strDataContext);
-			}
+                AppMDAO.saveStoreHits(appId.trim(), userId.trim(), tenantId);
```

Comment:
Prabath, I think you are referring to the 'AppMDAO' class name. We are to either change it to 'AppMDAO' or split this monolithic DAO class in to smaller classes.